src/less.rs

```diff
@@ -78,3 +95,28 @@ fn test_parse_less_version_wrong_program() {
 
     assert_eq!(None, parse_less_version(output));
 }
+
```

Comment:
Can you add this test too please? After making the code compile:

```
#[test]
fn test_parse_less_version_invalid_utf_8() {
    let output = b"\xff";

    assert_eq!(None, parse_less_version(output));
    assert_eq!(None, parse_less_version_busybox(output));
}
```

Also please add an assert for `parse_less_version_busybox()` to `test_parse_less_version_wrong_program()`.